bundles/ch.elexis.molemax/META-INF/MANIFEST.MF

```diff
@@ -1,5 +1,7 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
+Bundle-ClassPath: .,
+ lib/org.eclipse.nebula.widgets.gallery_1.0.0.202303072132.jar
 Bundle-Name: Molemax Plug-in für Elexis
 Bundle-SymbolicName: ch.elexis.molemax;singleton:=true
 Bundle-Version: 3.9.0.qualifier
```

Comment:
Du solltest die Version des Plugin erhöhen! Hier kommt neue Funktionalität rein,
mach 3.10.0.qualifier draus. Du musst dann auch enstprechend das feature in der Version erhöhen.